entity-framework/core/miscellaneous/cli/dotnet.md

```diff
@@ -1,43 +1,31 @@
 ---
 title: .NET Core CLI - EF Core
-author: rowanmiller
-ms.author: divega
-
-ms.date: 10/27/2016
-
-ms.assetid: c7c6824c-72be-4058-bdac-9b5b995b2f56
+author: bricelam
+ms.author: bricelam
+ms.date: 11/6/2017
 ms.technology: entity-framework-core
-
-uid: core/miscellaneous/cli/dotnet
 ---
-# .NET Core Command Line Tools for EF Core
-
-> [!IMPORTANT]  
-> The [.NET Core SDK](https://www.microsoft.com/net/download/core) no longer supports `project.json` or Visual Studio 2015. Everyone doing .NET Core development is encouraged to [migrate from project.json to csproj](https://docs.microsoft.com/dotnet/articles/core/migration/) and [Visual Studio 2017](https://www.visualstudio.com/downloads/).
-
-Entity Framework Core .NET Command Line Tools
-
-## Installation
-
-### Prerequisites
-
-.NET Command Line Tools require the .NET Core SDK. See the [.NET Core](https://www.microsoft.com/net/core) website for installation instructions.
-
-### Supported .NET platforms
+EF Core .NET Command Line Tools
+===============================
+The Entity Framework Core .NET Command Line Tools are an extension to the cross-platform **dotnet** command which is
+part of the [.NET Core SDK][2].
 
-The .NET Command Line Tools work on projects targeting .NET Core (e.g. "netcoreapp1.0", "netcoreapp1.1", "netcoreapp2.0", etc.). Use a version of the .NET CLI tools for EF Core package that matches the major version of the EF Core runtime used by the application.
+> [!TIP]
+> If you're using Visual Studio, we recommend the [the PMC Tools][1] instead since they provide a more integrated
+> experience.
 
-### Install by editing project
+Installing the tools
```

Comment:
So is it only if you install the package using NuGet that you then need to manually edit the csproj file? If so, should we call that out? Or am I missing something. :-)

Reply:
If you install EFCore.Tools.DotNet using `dotnet`, it will add it as a ProjectReference and you'll have to fix your project. VS throws if you try to install it there.

Reply:
I think mentioning any of that here will just confuse people. They really  just need to fix ~~dotnet/cli#5998~~ NuGet/Home#4190.